### ISSUE TITLE
Appveyor build fails

### ISSUE THREAD

**Issue by ysb33r** [ysb33r/gradleTest#6]
See this error at present
```
FAILURE: Build failed with an exception. 
 
* What went wrong:
A problem occurred configuring root project 'gradletest'.
> Failed to notify project evaluation listener.
   > C:\Users\appveyor\.gvm\gradle
 
* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output.
```

**Comment by ysb33r** [comment]
A more detailed stacktrace
```
org.ysb33r.gradle.gradletest.GradleTestPluginSpec > Effect of plugin on afterEvaluate FAILED 
    org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project 'gtp'.
        at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:79)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:74)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:61)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:470)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:79)
        at org.ysb33r.gradle.gradletest.GradleTestPluginSpec.Effect of plugin on afterEvaluate_closure2(GradleTestPluginSpec.groovy:54)
        at groovy.lang.Closure.call(Closure.java:423)
        at groovy.lang.Closure.call(Closure.java:439)
        at org.ysb33r.gradle.gradletest.GradleTestPluginSpec.Effect of plugin on afterEvaluate(GradleTestPluginSpec.groovy:48)
 
        Caused by:
        org.gradle.listener.ListenerNotificationException: Failed to notify project evaluation listener.
            at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:94)
            at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:31)
            at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
            at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:67)
            ... 7 more
 
            Caused by:
            java.io.FileNotFoundException: C:\Users\appveyor\.gvm\gradle
                at org.ysb33r.gradle.gradletest.internal.DistributionInternal.searchInstallFolder(DistributionInternal.groovy:85)
                at org.ysb33r.gradle.gradletest.internal.DistributionInternal.searchGvm(DistributionInternal.groovy:125)
                at org.ysb33r.gradle.gradletest.internal.DistributionInternal.searchLocal(DistributionInternal.groovy:146)
                at org.ysb33r.gradle.gradletest.GradleTestPlugin.apply_closure1_closure2(GradleTestPlugin.groovy:65)
                at groovy.lang.Closure.call(Closure.java:423)
                at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
                at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
                at org.gradle.listener.BroadcastDispatch.dispatch(BroadcastDispatch.java:83)
                ... 10 more
```

**Comment by ysb33r** [comment]
This problem has been solved, but see #9.
